worker/main.go

```diff
@@ -28,10 +28,13 @@ func main() {
 	}
 	completeQueue, _ := connection.OpenQueue(os.Getenv("SEND_TO_WHATSAPP_QUEUE"))
 	convertQueue, _ := connection.OpenQueue(os.Getenv("CONVERT_TO_WEBP_QUEUE"))
+
 	convert := &convert.ConvertConsumer{
 		// set to push to completeQueue when done
+		//set to push metrics to loggingQueue when done
```

Comment:
Comment is now stale as worker doesn't push metrics anymore no?

Reply:
Exactly